designs/2022-community-eslint-org/README.md

```diff
@@ -133,53 +133,71 @@ I personally think that the acceptance criteria for a package to go under the
 owner of the repo to transfer ownership over to the new org, we could
-temporarily fork the repo & publish the package as
-`@eslint-community-fork/<package name>` or so.
+temporarily fork the repo into the new org. However, the ultimate goal would
+always be to transfer the original repo to the new org.  
+If we had a temporary fork, we should create PRs to the original repo to bring
+it up to date with the fork before continuing with other tasks.
 
 ### What permissions should maintainers have?
 
-I think it would be best to have some sort of a "core" team (of at least 3(?)
-people) that's making the decisions in terms of acceptance to the repo, which
-would (in addition to ESLint TSC) all be admin of the org.
+I think it would be best to have some sort of a community core team (of at least
+3(?) people) that's making the decisions in terms of acceptance to the repo,
+which would (in addition to ESLint TSC) all be admins of the org.
+
+We can make a team (with sub-teams, if we want) for each package, where we give
+maintainer rights on the repo for that specific package, but keep admin rights
+to ESLint TSC & the community core team. This way we prevent that the repo would
+be destroyed from the org.
+
+We'll create a setup where we publish new versions of the package with
+`eslintbot`. That way we can keep publishing new versions if someone would
+dissapear.
 
-We can make a team (with sub-teams if we want) for each package, where we give
-admin rights on the repo for that specific package.
+The community core team would help maintain all repos, whereas people in
+specific teams would only help maintain these repositories.
 
-The "core" team would help maintain all repos, whereas people in specific teams
-would only help maintain these repositories.
+The community core team would be responsible of administer the Code of Conduct
+of the organization.
 
-The "core" team would also be responsible to contact package maintainers
+The community core team would also be responsible to contact package maintainers
 (outside of GitHub) if they appear inactive. If the package maintainers don't
 respond in a reasonable time frame (npm gives you 14(?) days to respond to a
-transfer request email), they could be considered non-responsive and the "core"
-team can step in to appoint a new maintainer for the repository.
+transfer request email), they could be considered non-responsive and the
+community core team can step in to appoint a new maintainer for the repository.
 
 **Bonus:** I think having our own place in the Discord server where we can
 discuss some things would be a good idea as well.  
-This way the "core" team has an open place to discuss some common things like
-extracting common functionality into a separate package, minimum Node & ESLint
-target, how to handle disputes between maintainers, ...  
+This way the community core team has an open place to discuss some common things
```

Comment:
Let’s be specific: is this a bonus or a must-have? Are we talking one channel? Is it private? What about a space for maintainers to collaborate? Does each package need a channel? Or can they request one?

Should this be a separate Discord server?

Reply:
For me personally, this is not a must-have, as we could operate the new org without it, but it's more like a "really nice to have".

I think it would be nice to keep this in the same Discord server, but under a separate group of channels, as that would keep all ESLint related discussions in 1 server and we can easily link people to the right channel that way.

Having the community core group responsible for creating these channels would be a good idea imo, as this way it won't create extra work for the ESLint TSC.

I think that having a separate channel for each package would be easiest for people to talk about things regarding that specific package. This also makes sure that these discussions don't pollute other packages' discussions.

Reply:
Can you update the TFC with this?

I think we should probably just start with one channel for the community core team and worry about whether packages need their own channels later. I’m guessing most won’t have an active need for it.